installation/linux.md

```diff
@@ -161,21 +159,53 @@ Yum or Dnf Based Systems
 
 {% include collapsible/body.html %}
 
-**Note:** The RPM files within the cannot currently be GPG signed or checked, only continue if you're happy installing without GPG verification.
+**Note:** The RPM files cannot currently be GPG signed or checked, only continue if you're happy installing without GPG verification.
 
-The RPM repo information should be defined inside a file, make a new file at `/etc/yum.repos.d/openhab.repo` with the following contents:
+The RPM repo information should be defined inside a file, make a new file at `/etc/yum.repos.d/openhab.repo` with the following contents depending on your choice of release.
+You may add all three to the same file, but make sure the desired repo is is set to `enabled=0`:
 
-```text
-[openHAB-Snapshots]
-name=openHAB 2.x.x Snapshots
-baseurl=https://openhab.jfrog.io/openhab/openhab-linuxpkg-rpm/unstable
-enabled=1
-gpgcheck=0
-repo_gpgcheck=0
-gpgkey=https://openhab.jfrog.io/openhab/api/gpg/key/public 
-```
+* **Official Release**
+
+  The stable builds contain the latest official release with tested features.
+
+  ```text
+  [openHAB-Release]
+  name=openHAB 2.x.x Release
```

Comment:
Is release contrary to testing? How about 

* Stable Release
* Testing Release
* Snapshot/Unstable Release